bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/admin/MaintenanceHandler.java

```diff
@@ -184,17 +197,53 @@ public Response deleteAll( @ApiParam( "The path to delete globally" ) final @Pat
     @Path( "/infinispan/cache/{name}" )
     @DELETE
     public Response cleanInfinispanCache(
-                    @ApiParam( "The name of cache to clean, 'all' for all caches" ) @PathParam( "name" ) final String name )
+                    @ApiParam( "The name of cache to clean" ) @PathParam( "name" ) final String name )
+    {
+        return Response.status( FORBIDDEN ).build();
```

Comment:
Some of our caches are still disposable though...I wonder if we want to have a ban list, or maybe a whitelist? I'm willing to be talked out of that position, but I'm inclined to think we should be able to clear certain caches...things like the koji metadata cache, for example.

Reply:
White list is better but hard to select and maintain... I will try a ban list, basically folo*, store*, and schedule-expire-cache.